R/retrieveData.R

```diff
@@ -43,8 +46,9 @@
 #' @importFrom methods formalArgs
 #' @importFrom utils sessionInfo tar modifyList
 #' @importFrom withr with_dir with_tempdir local_options
+#' @importFrom renv snapshot
 #' @export
-retrieveData <- function(model, rev = 0, dev = "", cachetype = "rev", puc = identical(dev, ""), strict = FALSE, ...) { # nolint
+retrieveData <- function(model, rev = 0, dev = "", cachetype = "rev", puc = identical(dev, ""), strict = FALSE, renv = TRUE, ...) { # nolint
```

Comment:
Having `renv` as a argument or variable name could be confusing because renv is already referring to the package, how about `runInRenv` or `useRenv` instead?